### PR TITLE
Improve form field sizing

### LINKED ISSUE #101: Handle custom input size with CSS to fix different behaviour across browsers
## Problem

It seems that implementation of `input` `size` attribute differs across browsers: `input`s are narrower in Chrome than in Firefox and Safari:

image

↑ Input with `size` of `16`, Chrome.

image

↑ The same field in Firefox.

This is [probably caused by vague specification of HTML](https://stackoverflow.com/a/27315464).

## Suggested Solution

There is probably no pure HTML + CSS solution, CSS cannot apply the value of `size` attribute anywhere else than in `content`.

However, we can take the value of `inputSize` and pass it to CSS through a custom property. Setting a width should then be as easy as this:

```css
width: calc(1ch * var(--rui-local-input-size));
```

Or probably more purely:

```css
width: var(--rui-local-input-size);
```

### LINKED ISSUE #102: Limit width of long form fields with manual `size`
Inputs should shrink automatically when their `size` is too big to fit their container.

image

### PR DESCRIPTION
**Pull request by adamkudrna**
- Limit width of form fields to prevent overflowing their container (closes #102)
- Handle custom input size with CSS to fix different behaviour across browsers (closes #101)

### CODE REVIEW

**Comment by bedrich-schindler** [comment]
I generally agree with this PR but I have few thing in my mind.

1. You changed `TextField` to accept input size. But there are other input fields so what about them?
2. There is no example in demo part of repository. Don't we want to show it it demo?

File: src/lib/components/ui/TextField/TextField.jsx [hunk 1]
```diff
@@ -65,6 +65,16 @@ export const TextField = (props) => {
     rootVariantClass = styles.rootVariantOutline;
   }
 
+  const inlineStyle = (inputSize) => {
```
Comment (bedrich-schindler) [inline src/lib/components/ui/TextField/TextField.jsx:68]: Test is missing for this feature

**Comment by adamkudrna** [comment]
> 1. You changed `TextField` to accept input size. But there are other input fields so what about them?

Only `TextField` accepts input size, the HTML `size` attribute is not available for any other form elements.

> 2. There is no example in demo part of repository. Don't we want to show it it demo?

Yes it is:
image
image

File: src/lib/components/ui/TextField/TextField.jsx (hunk 1 above)
Comment (adamkudrna) [reply src/lib/components/ui/TextField/TextField.jsx:68]: It's been already included in _all props_ test suite. Is it necessary to test it separately?

Comment (bedrich-schindler) [reply src/lib/components/ui/TextField/TextField.jsx:68]: Then is ok.

**Pull request merged**